lib/galaxy/jobs/runners/util/cli/shell/rsh.py

```diff
@@ -1,10 +1,14 @@
-from logging import getLogger
+import logging
+
+import paramiko
 
 from .local import LocalShell
+from ....util import Bunch
```

Comment:
This should probably just be ``galaxy.util import Bunch`` - not a big deal though.